docs/core/profiles.md

```diff
@@ -100,7 +100,7 @@ way tags. The available categories are specified by using the `graph.encoded_val
 you do further customization of GraphHopper) the possible categories are defined in `DefaultEncodedValueFactory.java`.
 For example there are these categories (some of their possible values are given in brackets).
 
-# todonow: should we try to list them all here?
+todonow: should we try to list them all here?
```

Comment:
We could try or link to the source code :) ?

Thanks for this documenation btw!!